tests/bwc/test_upgrade.py

```diff
@@ -571,14 +571,26 @@ def test_snapshot_compatibility(self):
         self._process_on_stop()
 
 
-class ReturningUpdateNodesCompatibilityTest(NodeProvider, unittest.TestCase):
+class ReturningNodesCompatibilityTest(NodeProvider, unittest.TestCase):
 
-    VERSIONS = ('4.1.0', '4.1.1', 'latest-nightly')
+
+    VERSIONS = ('4.1', 'latest-nightly')
```

Comment:
```suggestion
    VERSIONS = ('4.1.x', 'latest-nightly')
```

`4.1` results in building CrateDB from the branch, which is slower than using a released version.